lib/bundle/brewfile.rb

```diff
@@ -2,22 +2,22 @@
 
 module Bundle
   module Brewfile
+    ENV_KEY = "HOMEBREW_BUNDLE_FILE"
     module_function
 
     def path(dash_writes_to_stdout: false)
+      env_bundle_file = ENV.fetch(ENV_KEY, "")
+
       if ARGV.include?("--global")
```

Comment:
Could assign all the results to a variable and then `Pathname.new` that variable to avoid `Pathname.new` being repeated in each case.

Reply:
That makes sense. It looks like the `.expand_path(Dir.pwd)` can safely be added to `Pathname.new("#{ENV["HOME"]}/.Brewfile")`.